.github/actions/e2e/install-helm/action.yaml

```diff
@@ -0,0 +1,15 @@
+name: InstallHelm
```

Comment:
IIUC you could build a tools image (like the one we have) and run the container with gha so you have all your deps packaged up.

Reply:
Yep, you're right. Was planning to take care of it in a follow-up. Wanted to get the initial iteration out and then follow-up in another PR to add the image.